File: 10929 You can say 11/solution.java
import java.io.*;

public class Main {
	public static void main (String[] args) throws IOException {
		BufferedReader br = new BufferedReader(new InputStreamReader(System.in));
		String N;
		StringBuilder output = new StringBuilder();
		while((N = br.readLine()) != null && N.charAt(0) != '0') {
		    int sum = 0;
		    for(int i = 0; i < N.length(); ++i) {
    	        if((i & 1) == 1)
    	            sum -= N.charAt(i) - '0';
    	        else
    	            sum += N.charAt(i) - '0';
    	    }
    	    output.append(N).append(sum % 11 == 0 ? " is a multiple of 11.\n" : " is not a multiple of 11.\n");
		}
		System.out.print(output);
	}
}

File: 10929 You can say 11/solution.c
#include <stdio.h>
#include <string.h>

int main() {
	char N[1001];
	while(scanf("%s", N) && N[0] != '0') {
	    int sum = 0, len = strlen(N);
	    for(int i = 0; i < len; ++i) {
	        if(i & 1)
	            sum -= N[i] - '0';
	        else
	            sum += N[i] - '0';
	    }
	    printf("%s is %sa multiple of 11.\n", N, sum % 11 == 0 ? "" : "not ");
	}
	return 0;
}
